src/ppx_irmin/lib/deriver.ml

```diff
@@ -103,8 +103,7 @@ module Located (A : Ast_builder.S) : S = struct
                       rec_detected := true;
                       generic_name
                       (* If not a base type, assume a composite generic with the
-                         same naming convention *)
-                      )
+                         same naming convention *) )
```

Comment:
This one does not look very nice and is reported in https://github.com/ocaml-ppx/ocamlformat/issues/1228.